Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+
                 // Only show first tab with errors
                 if (!firstTabWithErrors) {
-                    tab.tab('show');
+
+                    tab.tab("show");
                     firstTabWithErrors = tab;
+
                 }
 
-                icon.removeClass('hide');
+                icon.removeClass("hide");
+
             } else {
-                icon.addClass('hide');
+
+                icon.addClass("hide");
+
             }
+
         });
+
     },
 
-    setup_inline_form_errors: function(subject) {
-        Admin.log('[core|setup_inline_form_errors] show first tab with errors', subject);
+    "setup_inline_form_errors" (subject) {
+
+        Admin.log("[core|setup_inline_form_errors] show first tab with errors", subject);
 
-        var deleteCheckboxSelector = '.sonata-ba-field-inline-table [id$="_delete"][type="checkbox"]';
+        const deleteCheckboxSelector = ".sonata-ba-field-inline-table [id$=\"_delete\"][type=\"checkbox\"]";
+
+        jQuery(deleteCheckboxSelector, subject).each(function () {
 
-        jQuery(deleteCheckboxSelector, subject).each(function() {
             Admin.switch_inline_form_errors(jQuery(this));
+
         });
 
-        jQuery(subject).on('change', deleteCheckboxSelector, function() {
+        jQuery(subject).on("change", deleteCheckboxSelector, function () {
+
             Admin.switch_inline_form_errors(jQuery(this));
+
         });
+
     },
 
     /**
      * Disable inline form errors when the row is marked for deletion
      */
-    switch_inline_form_errors: function(subject) {
-        Admin.log('[core|switch_inline_form_errors] switch_inline_form_errors', subject);
+    "switch_inline_form_errors" (subject) {
 
-        var row = subject.closest('.sonata-ba-field-inline-table'),
-            errors = row.find('.sonata-ba-field-error-messages')
-        ;
+        Admin.log("[core|switch_inline_form_errors] switch_inline_form_errors", subject);
 
-        if (subject.is(':checked')) {
-            row
-                .find('[required]')
-                .removeAttr('required')
-                .attr('data-required', 'required')
-            ;
+        let row = subject.closest(".sonata-ba-field-inline-table"),
+            errors = row.find(".sonata-ba-field-error-messages");
+        if (subject.is(":checked")) {
 
+            row.
+                find("[required]").
+                removeAttr("required").
+                attr("data-required", "required");
             errors.hide();
+
         } else {
-            row
-                .find('[data-required]')
-                .attr('required', 'required')
-            ;
 
+            row.
+                find("[data-required]").
+                attr("required", "required");
             errors.show();
+
         }
+
     },
 
-    setup_tree_view: function(subject) {
-        Admin.log('[core|setup_tree_view] setup tree view', subject);
+    "setup_tree_view" (subject) {
```

Comment:
- eslint: Expected object keys to be in ascending order. 'setup_tree_view' should be before 'switch_inline_form_errors'.